src/Raven.Server/ServerWide/Commands/CompareExchangeCommands.cs

```diff
@@ -12,10 +15,10 @@ namespace Raven.Server.ServerWide.Commands
     public abstract class CompareExchangeCommandBase : CommandBase
     {
         public string Key;
-        public string Database;
+        public string DatabaseName;
```

Comment:
Cannot be changed. Would brake backward compatibility and you would not be able to deserialize old compare exchange commands.

Reply:
done